### PR TITLE
style: Move prompt below user

### PR DESCRIPTION
**Pull request by tiagaoalb**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [ ] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description

Moved the prompt below the user name in spaceship theme

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/spaceship.omp.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
           "invert_powerline": false,
           "foreground": "lightYellow",
           "properties": {
-            "display_host": false,
+            "display_host": true,
             "prefix": "",
-            "user_info_separator": "",
+            "user_info_separator": "@",
```
Comment (JanDeDobbeleer) [inline themes/spaceship.omp.json]: this property has become obsolete, please use a `template` ;-)

File: themes/spaceship.omp.json [hunk 2]
```diff
@@ -16,9 +16,9 @@
           "invert_powerline": false,
           "foreground": "lightYellow",
           "properties": {
-            "display_host": false,
+            "display_host": true,
```
Comment (JanDeDobbeleer) [inline themes/spaceship.omp.json]: this property has become obsolete, please use a `template` ;-)

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: themes/spaceship.omp.json (hunk 2 above)
Comment (tiagaoalb) [reply themes/spaceship.omp.json]: Thank you! Can you check again pls?

Comment (tiagaoalb) [reply themes/spaceship.omp.json]: > 

Thank you! Can you check again pls?

**Pull request merged**